borbskeeper/ParseManager/BorbParseManager.m

```diff
@@ -106,6 +106,42 @@ + (void)loadMoreIncompleteTasksOfUser:(NSString *)username withLaterDate:(NSDate
     
 }
 
++ (void)fetchCompleteTasksOfUser:(NSString *)username WithCompletion:(void (^)(NSMutableArray *))completion {
+    PFQuery *query = [PFQuery queryWithClassName:QUERY_TASK_NAME];
+    query.limit = 20;
```

Comment:
let's make this a constant? We are using the same number multiple times

Reply:
Yea that's true; will do.